test/runtests.jl

```diff
@@ -21,53 +21,73 @@ const TRIXI_NTHREADS   = clamp(Sys.CPU_THREADS, 2, 3)
     # Hence, the additional flag `--compiled-modules=no` is required for Julia
     # versions older than v1.6.
     mpiexec() do cmd
-      run(`$cmd -n $TRIXI_MPI_NPROCS $(Base.julia_cmd()) --threads=1 --check-bounds=yes $(abspath("test_examples_2d_parallel.jl"))`)
+      run(`$cmd -n $TRIXI_MPI_NPROCS $(Base.julia_cmd()) --threads=1 --check-bounds=yes $(abspath("test_mpi.jl"))`)
     end
   end
 
-  @time if TRIXI_TEST == "all" || TRIXI_TEST == "2d_threaded"
+  @time if TRIXI_TEST == "all" || TRIXI_TEST == "threaded"
     # Do a dummy `@test true`:
     # If the process errors out the testset would error out as well,
     # cf. https://github.com/JuliaParallel/MPI.jl/pull/391
     @test true
 
-    run(`$(Base.julia_cmd()) --threads=$TRIXI_NTHREADS --check-bounds=yes --code-coverage=none $(abspath("test_examples_2d_parallel.jl"))`)
+    run(`$(Base.julia_cmd()) --threads=$TRIXI_NTHREADS --check-bounds=yes --code-coverage=none $(abspath("test_threaded.jl"))`)
   end
 
-  @time if TRIXI_TEST == "all" || TRIXI_TEST == "1d"
-    include("test_examples_1d.jl")
+  @time if TRIXI_TEST == "all" || TRIXI_TEST == "tree_part1"
+    include("test_tree_1d.jl")
+    include("test_tree_2d_part1.jl")
   end
 
-  @time if TRIXI_TEST == "all" || TRIXI_TEST == "2d_part1"
-    include("test_examples_2d_part1.jl")
+  @time if TRIXI_TEST == "all" || TRIXI_TEST == "tree_part2"
+    include("test_tree_2d_part2.jl")
   end
 
-  @time if TRIXI_TEST == "all" || TRIXI_TEST == "2d_part2"
-    include("test_examples_2d_part2.jl")
+  @time if TRIXI_TEST == "all" || TRIXI_TEST == "tree_part3"
+    include("test_tree_2d_part3.jl")
   end
 
-  @time if TRIXI_TEST == "all" || TRIXI_TEST == "2d_part3"
-    include("test_examples_2d_part3.jl")
+  @time if TRIXI_TEST == "all" || TRIXI_TEST == "tree_part4"
+    include("test_tree_3d_part1.jl")
   end
 
-  @time if TRIXI_TEST == "all" || TRIXI_TEST == "dgmulti"
-    include("test_examples_dgmulti.jl")
+  @time if TRIXI_TEST == "all" || TRIXI_TEST == "tree_part5"
+    include("test_tree_3d_part2.jl")
   end
 
-  @time if TRIXI_TEST == "all" || TRIXI_TEST == "3d_part1"
-    include("test_examples_3d_part1.jl")
+  @time if TRIXI_TEST == "all" || TRIXI_TEST == "tree_part6"
+    include("test_tree_3d_part3.jl")
```

Comment:
This would mean that we get rid of the additional dimensional structure completely, e.g.
```julia
@time if TRIXI_TEST == "all" || TRIXI_TEST == "tree_part1"
    include("test_tree_1d.jl")
    include("test_tree_2d_part1.jl")
  end
```
or
```julia
@time if TRIXI_TEST == "all" || TRIXI_TEST == "structured"
    include("test_structured_1d.jl")
    include("test_structured_2d.jl")
    include("test_structured_3d.jl")
  end
```
or
```julia
@time if TRIXI_TEST == "all" || TRIXI_TEST == "unstructured_dgmulti"
    include("test_unstructured_2d.jl")
    include("test_dgmulti_2d.jl")
    include("test_dgmulti_3d.jl")
  end
```
Is that desired?

Reply:
Ah, no, of course not. That sounds awful. My bad.